node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -2885,31 +3052,80 @@ mod tests {
         let subject = make_subject(&root_node_cryptde);
         let before = time_t_timestamp();
+        let mut neighborhood_metadata = make_default_neighborhood_metadata();
+        neighborhood_metadata.user_exit_preferences_opt = Some(UserExitPreferences {
+            exit_countries: vec!["CZ".to_string()],
+            fallback_preference: FallbackPreference::ExitCountryWithFallback,
+            locations_opt: Some(vec![ExitLocation {
+                country_codes: vec!["CZ".to_string()],
+                priority: 1,
+            }]),
+            db_countries: vec!["CZ".to_string()],
+        });
 
         let result = subject.handle(
             &mut dest_db,
             gossip.try_into().unwrap(),
             gossip_source,
-            make_default_neighborhood_metadata(),
+            neighborhood_metadata,
         );
 
         let after = time_t_timestamp();
-        assert_eq!(GossipAcceptanceResult::Accepted, result);
+
+        let expected = "Reply(Gossip_0v1 { node_records: [
+GossipNodeRecord {
+	inner: NodeRecordInner_0v1 {
+		public_key: 0x01020304,
+		node_addr_opt: Some(1.2.3.4:[1234]),
+		earning_wallet: Wallet { kind: Address(0x546900db8d6e0937497133d1ae6fdf5f4b75bcd0) },
+		rate_pack: RatePack { routing_byte_rate: 1235, routing_service_rate: 1434, exit_byte_rate: 1237, exit_service_rate: 1634 },
+		neighbors: [0x02030405],
+		version: 1,
+	},
+	node_addr_opt: Some(1.2.3.4:[1234]),
+	signed_data:
+Length: 254 (0xfe) bytes
+0000:   a8 6a 70 75  62 6c 69 63  5f 6b 65 79  44 01 02 03   .jpublic_keyD...
+0010:   04 6e 65 61  72 6e 69 6e  67 5f 77 61  6c 6c 65 74   .nearning_wallet
+0020:   a1 67 61 64  64 72 65 73  73 94 18 54  18 69 00 18   .gaddress..T.i..
+0030:   db 18 8d 18  6e 09 18 37  18 49 18 71  18 33 18 d1   ....n..7.I.q.3..
+0040:   18 ae 18 6f  18 df 18 5f  18 4b 18 75  18 bc 18 d0   ...o..._.K.u....
+0050:   69 72 61 74  65 5f 70 61  63 6b a4 71  72 6f 75 74   irate_pack.qrout
+0060:   69 6e 67 5f  62 79 74 65  5f 72 61 74  65 19 04 d3   ing_byte_rate...
+0070:   74 72 6f 75  74 69 6e 67  5f 73 65 72  76 69 63 65   trouting_service
+0080:   5f 72 61 74  65 19 05 9a  6e 65 78 69  74 5f 62 79   _rate...nexit_by
+0090:   74 65 5f 72  61 74 65 19  04 d5 71 65  78 69 74 5f   te_rate...qexit_
+00a0:   73 65 72 76  69 63 65 5f  72 61 74 65  19 06 62 69   service_rate..bi
+00b0:   6e 65 69 67  68 62 6f 72  73 81 44 02  03 04 05 73   neighbors.D....s
+00c0:   61 63 63 65  70 74 73 5f  63 6f 6e 6e  65 63 74 69   accepts_connecti
+00d0:   6f 6e 73 f5  6b 72 6f 75  74 65 73 5f  64 61 74 61   ons.kroutes_data
+00e0:   f5 67 76 65  72 73 69 6f  6e 01 70 63  6f 75 6e 74   .gversion.pcount
+00f0:   72 79 5f 63  6f 64 65 5f  6f 70 74 62  41 55         ry_code_optbAU
+	signature:
+Length: 24 (0x18) bytes
+0000:   01 02 03 04  f7 26 1e a5  d4 4a 71 d9  f8 42 08 35   .....&...Jq..B.5
+0010:   e4 99 27 4a  85 7f 01 11                             ..'J....
+}] }, 0x02030405, 2.3.4.5:[2345])".to_string();
```

Comment:
Tady si nejsem jisty, proc musis uvadet cela ta data. 

Mel bys byt schopen zkonstruovat ten expected GossipAcceptanceResult, ktery implementuje PartialEq + Debug, takze by nemel byt problem to pote porovnat s tim skutecnym vysledkem. 

Zkus to prosim. Tohle zde nahore je zbytecne obrovske a hlavne se to bude menit a kazit assertion s kazdou drobounkou zmenou v tom nasem protokolu. Mnohem lepsi je pracovat v assertions primo s temi datovymi strukturami.